TypewiseAlert.Test/TypewiseAlertTest.cs

```diff
@@ -8,8 +9,34 @@ public class TypewiseAlertTest
     [Fact]
     public void InfersBreachAsPerLimits()
     {
-      Assert.True(TypewiseAlert.inferBreach(12, 20, 30) ==
-        TypewiseAlert.BreachType.TOO_LOW);
+      Assert.True(TypewiseAlert.inferBreach(12, 20, 30) is LowBreachType);
```

Comment:
Please include boundary conditions also in the tests. ex. What is the breachtype when value is 20 or 30?